src/App.js

```diff
@@ -1,43 +1,61 @@
+import { useEffect, useState } from "react";
 import "./App.css";
 import logo from "./logo.png";
-
+import React from "react";
+import Mockman from "mockman-js";
+import ProductsPage from "./pages/products-page/ProductsPage";
+import HomePage from "./pages/home-page/HomePage";
+import CartPage from "./pages/cart-page/CartPage";
+import WishList from "./pages/wishlist-page/WishList";
+import LoginPage from "./pages/login-page/LoginPage";
+import SignupPage from "./pages/signup-page/SignupPage";
```

Comment:
you can use barrel imports for different pages

Reply:
Yes Same will be done in next PR, thanks